mission/functions/core/teams/fn_force_team_change.sqf

```diff
@@ -20,13 +20,13 @@ params ["_player", "_team"];
 
 private _playerGroup = _player getVariable ["vn_mf_db_player_group", "FAILED"];
 private _playerGroupArray = missionNamespace getVariable [_playerGroup,[]];
-private _playerUID = getPlayerUID _player;
 private _isWhitelisted = [_player, _team] call para_g_fnc_db_check_whitelist;
 
-if (_isWhitelisted) then {
-	_player setVariable ["vn_mf_db_player_group", _team, true]; 
-} else {
-	_player setVariable ["vn_mf_db_player_group", "MikeForce", true]; 	
+if !(_isWhitelisted) then {
+
+	private _shortname = getText (missionConfigFile >> "gamemode" >> "teams" >> _team >> "shortname");
+	["ErrorNotWhitelistedForTeam", [_shortname]] remoteExec ["para_c_fnc_show_notification", _player];
+
```

Comment:
Show a notification to players when they are not whitelisted for a team

Reply:
!todo: Switch to last known team from `_playerGroup` variable (if `FAILED` then switch to `MikeForce`)